pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
Yes, this seems like the right approach. Currently I'm defaulting to Parallel when for whatever reason pool doesn't exist but n_jobs > 1 (i think maybe you unpickle existing models in some of the tests? is that possible?).